exporter/exporter.go

```diff
@@ -85,6 +85,16 @@ func LoadMeasuresAndValueSets(measures []byte, valueSets []byte) {
 
 // GenerateCat1 generates a cat1 xml string for export
```

Comment:
Overall this PR looks good. Is it possible you can revise the comments on `GenerateCat1` so that the interplay between it and `generateCat1` is more explicit?